CondFormats/PCLConfig/interface/AlignPCLThresholdsHG.h

```diff
@@ -0,0 +1,50 @@
+#ifndef CondFormats_PCLConfig_AlignPCLThresholdsHG_h
+#define CondFormats_PCLConfig_AlignPCLThresholdsHG_h
+
+#include "CondFormats/PCLConfig/interface/AlignPCLThresholds.h"
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include <map>
+#include <string>
+#include <vector>
+
+class AlignPCLThresholdsHG : public AlignPCLThresholds {
+public:
+  AlignPCLThresholdsHG() {}
+
+  enum FloatParamIndex {
+    FRACTION_CUT_X = 0,
+    FRACTION_CUT_Y = 1,
+    FRACTION_CUT_Z = 2,
+    FRACTION_CUT_TX = 3,
+    FRACTION_CUT_TY = 4,
+    FRACTION_CUT_TZ = 5,
+    FSIZE = 6
+  };
+
+  void setAlignPCLThresholdHG(const std::string &AlignableId, const std::vector<float> &floatVec);
+  void setAlignPCLThresholdsHG(const std::unordered_map<std::string, std::vector<float>> &floatMap_);
+
+  void SetFractionCut(const std::string &AlignableId, const coordType &type, const float &cut);
+
+  const std::unordered_map<std::string, std::vector<float>> &getFloatMap() const { return floatMap; }
+  std::vector<float> getFloatVec(const std::string &AlignableId) const;
```

Comment:
method getFloatVec should be possibly better return a const std::vector<float>& pointing to the corresponding const iterator...